test/_e2e/cmd_invoke_test.go

```diff
@@ -57,7 +57,7 @@ func TestInvoke(t *testing.T) {
 	)
 	defer cleanup()
 
-	run(t, bin, prefix, "create", "--language=go", "--template=cloudevents", cwd)
+	run(t, bin, prefix, "create", "--language=node", "--template=cloudevents", cwd)
 	set(t, "handle.go", TestInvokeFunctionImpl)
 	run(t, bin, prefix, "deploy", "--registry", GetRegistry())
```

Comment:
Hi @lkingland . Just an idea, since here the goal is to test invoke, the builder used to generate the function itself won't matter much (IMO) so probbly we could force it to use `pack` so it won't fail on midstream

```suggestion
	run(t, bin, prefix, "deploy", "--builder", "pack", "--registry", GetRegistry())
```
So you don't need to change the whole test implementation, wdyt?